md.js

```diff
@@ -2,6 +2,15 @@ var showdown = require('showdown');
 var converter = new showdown.Converter();
 
 exports.translate = function(load) {
+  var converterOptions = this.mdOptions;
```

Comment:
Instead of `this.mdOptions` I think we can use `load.metadata.mdOptions` here to use the metadata pipeline.